api/db-cdr/generate-cdr/csv/achilles_analysis.csv

```diff
@@ -6,9 +6,10 @@ analysis_id,"analysis_name","stratum_1_name","stratum_2_name","stratum_3_name","
 4,"Number of persons by race","race_concept_id","","","","","pie","counts"
 5,"Number of persons by ethnicity","ethnicity_concept_id","","","","","column","counts"
 6,"Number of persons by age decile","age_decile","","","","","column","counts"
+7,"Number of persons by gender_identity","gender_identity_concept_id","","","","","column","counts"
```

Comment:
Sorry, basic question, but what's the significance of the first column here? Is this just a database ID that you're choosing yourself (incrementally)?

Reply:
yeah, the auto id we are just adding when new row is added